docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -0,0 +1,119 @@
+
+- JetBrains IDE like PyCharm, IntelliJ, Goland etc. [Products](https://www.jetbrains.com/products/#type=ide)
+- One or more Teleport Nodes with Server Access enabled. If you have not yet
+done this, read the [Server Access Getting Started Guide](../getting-started.mdx) to learn how.
+
+## Step 1/3. First-time setup
+
+Configure your local SSH client to access Teleport Nodes, assigning the `--proxy` flag to the address of your Teleport Proxy Service (e.g., `mytenant.teleport.sh` for Teleport Cloud users).
+
+```code
+# log in to your proxy:
+$ tsh login --proxy proxy.foo.example.com --user alice
+
+# generate the OpenSSH config for the proxy:
+$ tsh config --proxy proxy.foo.example.com
+```
+
+Append the resulting configuration snippet into your SSH config file located
+in the path below:
+
+<Tabs>
+  <TabItem label="Linux/MacOS">
+    `$HOME/.ssh/config`
+  </TabItem>
+  <TabItem label="Windows">
+    `%UserProfile%\.ssh\config`
+
+    <Admonition type="warning">
+      If using PowerShell on Windows to write your SSH config, note that normal
+      shell redirection may write the file with the incorrect encoding. To
+      ensure it's written properly, try the following:
+
+      ```code
+      $ tsh.exe config | out-file .ssh\config -encoding utf8 -append
+      ```
+    </Admonition>
+  </TabItem>
+</Tabs>
+
+You should be able to connect to the desired node using following command, replacing `user` with the username you would like to assume on the node.
+
+```code
+$ ssh user@[node name].[cluster name]
+```
+
+<Details scopeOnly={true} scope={["cloud"]} title="Teleport Cloud">
+  The SSH config you generated earlier instructs your SSH client to run `tsh proxy ssh` to access a Node in your Teleport cluster.
+  However, running an `ssh` command against the Teleport Proxy Service at `yourtenant.teleport.sh` will result in an error.
+</Details>
+
+<Admonition type="note">
+  Teleport's certificates expire fairly quickly, after which SSH
+  attempts will fail with an error like the following:
+
+  ```txt
+  alice@proxy.foo.example.com: Permission denied (publickey).
+  ERROR: exit status 255
+
+  kex_exchange_identification: Connection closed by remote host
+  ```
+
+  When you see this error, re-run `tsh login` to refresh your local certificate.
+</Admonition>
+
+## Step 2/3. Configure JetBrains IDE
+
+After opening your IDE go to `Tools` -> `Deployment` -> `Browse Remote Host`.
+
+![Deployment](../../../img/jetbrains-sftp/deployment-main.png)
+
+Then click plus sign in the left top corner to add a new server.
+
+![Add server](../../../img/jetbrains-sftp/add-server.png)
+
+Enter a name for your new server.
+
+![New Deployment](../../../img/jetbrains-sftp/deployment-added.png)
+
+Click three dots next to `SSH configuration` as in the picture above.
+
+![SSH Configuration](../../../img/jetbrains-sftp/ssh-configurations.png)
+
+Create a new configuration by clicking plus sign on the left top and providing:
```

Comment:
I think the items in this list should use ` - ` and "is" consistently to demarcate fields from their descriptions. Personally, I think using "is" instead of ` - ` works a bit better here.